Test OIDC login controller (#9923)
18F/identity-idp: docs/local-development.md

To simulate a true end-to-end user experience, you can either...

- Use the built-in test controller for SAML logins at http://localhost:3000/test/saml/login or OIDC logins at http://localhost:3000/test/oidc/login

>   Note: to update service provider configurations,  run __rake db:seed__ comand

Reviewer: ```suggestion
  Note: to update service provider configurations,  run the command `rake db:seed` or `make setup`
```

`make setup` also runs `db:seed` so I think this note could be helpful.

Author: @night-jellyfish , thanks, I usually only use `make update`.
